catalogd/Makefile

```diff
@@ -118,9 +118,3 @@ build-container: build-linux ## Build docker image for catalogd.
 kind-load: $(KIND) ## Load the built images onto the local cluster
 	$(CONTAINER_RUNTIME) save $(IMAGE) | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
 
-##@ Deploy
```

Comment:
There's apparently no Deploy section in the op-con Makefile help. :(

Reply:
I think the demo should be under the doc section as it is right?
Should we create a Deploy section
Does it make sense we add it there?